references.go

```diff
@@ -133,20 +136,13 @@ func parentsContainingPath(path string, c *object.Commit) ([]*object.Commit, err
 	}
 }
 
-// Returns an slice of the commits in "cs" that has the file "path", but with different
-// contents than what can be found in "c".
-func differentContents(path string, c *object.Commit, cs []*object.Commit) ([]*object.Commit, error) {
-	result := make([]*object.Commit, 0, len(cs))
-	h, found := blobHash(path, c)
+func sameContent(path string, c1, c2 *object.Commit) (bool, error) {
+	h, found := blobHash(path, c1)
 	if !found {
-		return nil, object.ErrFileNotFound
-	}
-	for _, cx := range cs {
-		if hx, found := blobHash(path, cx); found && h != hx {
-			result = append(result, cx)
-		}
+		return false, object.ErrFileNotFound
```

Comment:
Does this mean that if 1 parent had the file and the other didn't it will error out with file not found?

Reply:
Ahh, I just realised that we are only going through the parents that we already confirmed contained the file